definitions/features/foreman_proxy.rb

```diff
@@ -144,7 +144,7 @@ def curl_cmd
 
     cmd = "curl -w '\n%{http_code}' -s "
     cmd += format_shell_args('--cert' => ssl_cert, '--key' => ssl_key, '--cacert' => ssl_ca)
-    cmd += " https://$(hostname):#{proxy_settings[:https_port]}"
+    cmd += " https://$(hostname -f):#{proxy_settings[:https_port]}"
```

Comment:
```suggestion
    cmd += " https://#{hostname}:#{proxy_settings[:https_port]}"
```

We have a helper for that! https://github.com/theforeman/foreman_maintain/blob/4292fdd670ab6eb739077641a66b3417eaee6356/lib/foreman_maintain/concerns/system_helpers.rb#L94-L96

Reply:
I completely missed that somehow. Fixed it now.

It now shows up like this:

~~~
DEBUG -- : Running command curl -w '
%{http_code}' -s  --cert /etc/foreman-proxy/foreman_ssl_cert.pem --key /etc/foreman-proxy/foreman_ssl_key.pem --cacert /etc/foreman-proxy/foreman_ssl_ca.pem https://smart-proxy.example.com:9090/features with stdin nil
~~~